src/main/kotlin/no/nav/familie/ef/søknad/api/RegistrerArbeidsaktivitetController.kt

```diff
@@ -18,7 +19,10 @@ import java.time.LocalDateTime
 
 @RestController
 @RequestMapping(path = ["/api/registrerarbeid"], produces = [APPLICATION_JSON_VALUE])
-@ProtectedWithClaims(issuer = EksternBrukerUtils.ISSUER, claimMap = ["acr=Level4"])
+@RequiredIssuers(
+    ProtectedWithClaims(issuer = "tokenx", claimMap = ["acr=Level4"]),
```

Comment:
Hvorfor ikke bruke `EksternBrukerUtils.X` ?

Reply:
Synes det ble mer forståelig med hardkoding, da det tidligere bare het ISSUER, så man fikk ikke sett om det var snakk om token_x eller selvbetjening. Nå har vi renamet de to variablene til bedre navn, så da synes jeg det er greit å bytte til EksternBrukerUtils igjen. Jeg fikser.

Reply:
Synes det var uklart hvilken issuer det var snakk om når det bare het ISSUER, så hardkodet dette. Nå har vi renamet det til bedre navn, så da kan jeg endre til å bruke det i egen PR hvor felles-versjon også oppgraderes